main.yml

```diff
@@ -431,7 +433,7 @@ openshift:
       - id: overview
         title: Overview
         group: openshift
-  source_repo: 
+  source_repo:
```

Comment:
What's up with this change?

Reply:
I mean it's probably fine though tbh

Reply:
Ah, probably my linter got it by accident. But yeah should be fine 👍